backend/Auth/ResetPassword.ts

```diff
@@ -115,6 +119,63 @@ router.get<Params, ResponseBody, QueryParams>(
   }
 );
 
+router.put(
+  "/api/passwords",
+  body("password").isStrongPassword(),
+  body("token").exists(),
+  async (req, res) => {
+    const { password, token } = req.body;
```

Comment:
This line might throw an error if the user makes a PUT request without a body. For example, the following code in the browser console would likely cause an error:

```js
debugFetch('/api/passwords', {method: 'PUT'})
```

The validations you put in on line 129 will catch this problem of body-less requests, but since this line is before line 129 that won't help us here.

To fix, just move this after the `if (!errors.isEmpty()) {` statement on line 130-142